week-8/Homework/mandatory/4-programmer-humour/script.js

```diff
@@ -0,0 +1,25 @@
+function load(){
+    fetch("https://xkcd.now.sh/?comic=latest")
+    .then( data => data.json())
+    .then( data => {
+        console.log(data)
+        const img = document.createElement('img')
+        img.src=data.img
+        const body = document.querySelector('#body')
```

Comment:
Reminder: You can use any CSS selector here, so you actually don't need to assign an ID to the body tag and look it up by ID - how could you write this line of code _not_ assigning an ID to the tag?